packages/react-core/src/operations/aggregation.js

```diff
@@ -24,6 +34,16 @@ export function aggregate(feature, keys, operation) {
   return aggregationFn(keys.map((column) => feature[column]));
 }
 
+function filterFalsyElements(values, keys) {
+  const filterFn = (value) => value !== null && value !== undefined;
+
+  if (!keys?.length) {
+    return values.filter(filterFn);
+  } else if (keys.length === 1) {
```

Comment:
can / should this be an 'else', not else if ?

Or even... can we remove the elseif and just return ?